energyplus_regressions/runtests.py

```diff
@@ -497,7 +497,7 @@ def both_files_exist(base_path_a, base_path_b, common_relative_path):
     @staticmethod
     def diff_perf_log(file_a, file_b, diff_file):
         # will do a pretty simple CSV text token comparison, no numeric comparison, and omit some certain patterns
-        tokens_to_skip = [1, 2, 29, 30]
+        tokens_to_skip = [1, 2, 30, 31]
```

Comment:
The new steam token in the perflog meant we needed to bump these last two indices.